Sensor.cpp

```diff
@@ -0,0 +1,31 @@
+#include <Sensor.h>
+
+namespace LFRobot
+{
+	Sensor::Sensor(int inPin)
+	{
+		pinMode(pin, INPUT);
```

Comment:
careful, you use `pin` before you initialize it.
It's best to use an initializer list to guarantee the variable is initialized before you use it